libs/exavmlib/lib/Code.ex

```diff
@@ -1,7 +1,7 @@
 #
 # This file is part of AtomVM.
 #
-# Copyright 2021 Davide Bettio <davide@uninstall.it>
+# Copyright 2024 Davide Bettio <davide@uninstall.it>
```

Comment:
No need to change this date here, I didn't do any additional work so far ;)

Reply:
Lol, needed a change to force-push the commit message change;-) - fixed.